DataConnectors/CohesitySecurity/Helios2Sentinel/IncidentProducer/IncidentProducer.cs

```diff
@@ -0,0 +1,215 @@
+using System.Dynamic;
+using Newtonsoft.Json.Converters;
+using Microsoft.WindowsAzure.Storage;
+using Helios2Sentinel;
+
+namespace Helios2Sentinel
+{
+    public class IncidentProducer
+    {
+        private static readonly object queueLock = new object();
+        private static Lazy<ConnectionMultiplexer> lazyConnection = CreateConnection();
+        public static long GetPreviousUnixTime(ILogger log)
+        {
+            DateTime previousDateTime = DateTime.Now;
+            try
+            {
+                previousDateTime = previousDateTime.AddDays(long.Parse(Environment.GetEnvironmentVariable("startDaysAgo")));
+            }
+            catch (Exception ex)
+            {
+                previousDateTime = previousDateTime.AddDays(-30);
+                log.LogError("Exception --> 1 " + ex.Message);
+            }
+            return ((DateTimeOffset)previousDateTime).ToUnixTimeMilliseconds() * 1000;
+        }
+
+        public static long GetCurrentUnixTime()
+        {
+            return ((DateTimeOffset)DateTime.Now).ToUnixTimeMilliseconds() * 1000;
+        }
+
+        public static ConnectionMultiplexer Connection
+        {
+            get
+            {
+                return lazyConnection.Value;
+            }
+        }
+
+        private static Lazy<ConnectionMultiplexer> CreateConnection()
+        {
+            return new Lazy<ConnectionMultiplexer>(() =>
+            {
+                return ConnectionMultiplexer.Connect(Environment.GetEnvironmentVariable("connectStr"));
+            });
+        }
+
+        public static async Task ParseAlertToQueueAsync(
+            [Queue("%CohesityQueueName%"), StorageAccount("AzureWebJobsStorage")] ICollector<string> outputQueueItem,
+            dynamic alert, ILogger log)
+        {
+            dynamic output = new ExpandoObject();
+            output.properties = new ExpandoObject();
+            output.properties.Description = alert.alertDocument.alertDescription + ". Alert cause: " + alert.alertDocument.alertCause + ". Additional Info: " + alert.alertDocument.alertHelpText + ". Helios ID: " + alert.id;
+            output.properties.title = "Cluster: " + alert.clusterName;
+            output.properties.status = ((string)alert.alertState).Equals("kOpen", StringComparison.OrdinalIgnoreCase) ? "New" : "Closed";
+            output.properties.severity = "Medium";
+
+            string id = alert.id;
+            int i = 0;
+
+            foreach (var prop in alert.propertyList)
+            {
+                switch (i)
+                {
+                case 12:
+                    long sev = long.Parse((string)prop.value);
+                    if (sev >= 70)
+                        output.properties.severity = "High";
+                    else if (sev < 30)
+                        output.properties.severity = "Low";
+                    else
+                        output.properties.severity = "Medium";
+                    break;
+                case 0:
```

Comment:
**Nit**: Sort the case statements in ascending order for a better readability, like this
                case 0:
                case 4:
                case 7:
                **case 10:**
                case 11: